draft-vvv-webtransport-quic.md

```diff
@@ -110,10 +112,10 @@ beneficial.
 
 When a client requests a QuicTransport to be created, the user agent establishes
 a QUIC connection to the specified address.  It verifies that the the server is
-a QuicTransport endpoint using ALPN, and that the client is allowed to connect
-to the specified endpoint using `web_accepted_origins` transport parameter.
-Once the verification succeeds and the QUIC connection is ready, the client can
-send and receive streams and datagrams.
+a QuicTransport endpoint using ALPN, and sends a *client indication* containing
```

Comment:
Is endpoint a defined term in IETF land?

Reply:
Well, quic-transport-23 defines it and uses it 460 times.